modules/exploits/multi/http/cve_2021_35464_forgerock_openam.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::JavaDeserialization
```

Comment:
This mixin doesn't appear to be used anywhere since the `Msf::Util` method is used directly. It should be safe to remove this.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/15386/commits/8b3d057e9e25c3f340962e5fd906052145a26e24